upstairs/src/lib.rs

```diff
@@ -3815,9 +3849,14 @@ async fn up_listen(
          * a upstairs generated flush.
          */
         let mut flush_check = deadline_secs(5);
+        let mut show_work_interval = deadline_secs(5);
 
         loop {
             tokio::select! {
+                _ = sleep_until(show_work_interval) => {
```

Comment:
We probably don't want this always on, but go ahead and leave this for now and perhaps when the dtrace stuff comes in we might turn this off or make it optional.